core-js/src/main/javascript/cdf/components/MultiButtonComponent.js

```diff
@@ -143,7 +143,7 @@ define([
           if(!myself.isMultiple) { break; }
         }
       }
-      if(((!foundDefault && !myself.isMultiple) || (!foundDefault && myself.isMultiple && myself.defaultIfEmpty))
+      if(((!foundDefault && !myself.isMultiple && myself.useFirstValue) || (!foundDefault && myself.isMultiple && myself.useFirstValue))
```

Comment:
Why did you change the first part to `(!foundDefault && !myself.isMultiple && myself.useFirstValue)`?
The idea is to simply replace the `defaultIfEmpty` with `useFirstValue`, this extra validation breaks the unit tests.

Reply:
because without this changes first value selected